CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/NewGraph.java

```diff
@@ -120,19 +124,20 @@ public void callService(final PluginParameters parameters, final InputStream in,
 
         try {
             newId = RestServiceUtilities.waitForGraphChange(existingId).get(10, TimeUnit.SECONDS);
+            if (!newId.isBlank()) {
+                final ObjectMapper mapper = new ObjectMapper();
+                final ObjectNode root = mapper.createObjectNode();
+                root.put("id", newId);
+                root.put("name", GraphNode.getGraphNode(newId).getDisplayName());
+                root.put("schema", schemaName);
+                mapper.writeValue(out, root);
+            }
         } catch (final InterruptedException ex) {
             Thread.currentThread().interrupt();
+            LOGGER.log(Level.SEVERE, "Thread interrupted");
```

Comment:
Are you able to add the exception as an argument here (just add ex as a 3rd parameter)